src/main/kotlin/org/rust/lang/core/psi/ext/RsStructLiteralField.kt

```diff
@@ -7,13 +7,15 @@ package org.rust.lang.core.psi.ext
 
 import com.intellij.lang.ASTNode
 import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RsFieldDecl
 import org.rust.lang.core.psi.RsStructLiteral
 import org.rust.lang.core.psi.RsStructLiteralField
 import org.rust.lang.core.resolve.ref.RsReference
 import org.rust.lang.core.resolve.ref.RsStructLiteralFieldReferenceImpl
 
 
 val RsStructLiteralField.parentStructLiteral: RsStructLiteral get() = parentOfType<RsStructLiteral>()!!
+val RsStructLiteralField.declaration: RsFieldDecl? get() = reference.resolve() as? RsFieldDecl
```

Comment:
Let's call it `resolveToDeclaration` (compare with `RsTraitItem#resolveToTrait`)?